packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -55,20 +50,17 @@ async function runPipelineLoadStep(
     if (!step.load) {
       continue;
     }
-    const pluginLogger =
-      pluginLoggers[step.name] || createLogger({name: step.name, level: logLevel});
-    pluginLoggers[step.name] = pluginLogger;
 
     try {
       const debugPath = path.relative(process.cwd(), srcPath);
-      pluginLogger.debug(`load() starting: [${debugPath}]`);
+      logger.debug(`load() starting… [${debugPath}]`, {name: step.name});
```

Comment:
In our logs, we have things like:

```
[snowpack] a generic thing happened
[@snowpack/plugin-babel] a Babel plugin thing happened
```

This 2nd param: `{ name: [something] }` changes the name at the beginning of the line. In the context of our build system, we find this helpful.